html/about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap">
+    <link rel="stylesheet" href="../css/all.min.css">
+    <link rel="stylesheet" href="../css/mainLanding.css">
+    <link rel="stylesheet" href="../css/about.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- start Header -->
+    <header>
+        <section class="menu-sectoin">
+            <div class="container">
+                <a href="../index.html"><img class="logo" src="../image/logo.jpg" alt="logo"></a>
+                <div class="burger-button">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+                <ul>
+                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="about.html" class="active">About</a></li>
+                    <li><a href="#">Sites</a></li>
+                    <li><a href="#">News</a></li>
+                </ul>
+                <button>Hygiene ambassadors</button>
+            </div>
+        </section>
+    </header>
+    <!-- End Header -->
+    <!-- Start landing -->
+    <section class="landing">
+        <div class="container">
+            <q>cleaning rivers</q>
+            <h1>hygiene ambassadors</h1>
+            <p>A youth campaign to clean the banks of rivers throughout Iraq</p>
+            <p class="contact-text">please contact us per Emeail for any further questions</p>
+            <p class="contact">bfhetyrhyr12@gmail.com</p>
+        </div>
+    </section>
+    <!-- End landing -->
+    <!-- Start logo-section -->
+    <section class="logo-section">
+        <div class="container">
+            <h2 class="text-section">Hygiene ambassadors Logo 2022</h2>
+            <p class="contact">The logo for the GameDev Academy was made by <br>
+            bfhetyrhyr12@gmail.com</p>
+            <img src="../image/logo.jpg" alt="">
+        </div>
+    </section>
+    <!-- end logo-section -->
+    <!-- Start best-years -->
+    <section class="best-years">
+        <div class="container">
+            <h2 class="text-section">See The Past Hygiene ambassadors Years</h2>
+            <p>Take a look at the past 2 years of the Hygiene ambassadors years.</p>
+    
+            <div class="box-container">
+                <div class="box">
+                    <h3>2021 <br>Hygiene ambassadors year 2021</h3>
+                </div>
+                <div class="box">
+                    <h3>2022 <br>Hygiene ambassadors year 2022</h3>
+                </div>
+            </div>
```

Comment:
- [ ] This section contains two images, please kindly add them 😉 😉. You should make sure to stick as much as possible to the original designer (eg: color, font,...)🙃 🙃 

Your design

<img width="689" alt="Screen Shot 2022-10-19 at 18 03 46" src="https://user-images.githubusercontent.com/74501523/196744805-bd23327e-22ec-424c-bc8a-d899c9250d61.png">

The original design

<img width="655" alt="Screen Shot 2022-10-19 at 18 04 07" src="https://user-images.githubusercontent.com/74501523/196744863-e3006813-b283-4efe-b02c-b59426146b22.png">